<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_28">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTabWidget" name="tabWidget">
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape">
        <enum>QTabWidget::Triangular</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="usesScrollButtons">
        <bool>true</bool>
       </property>
       <widget class="PollDeviceWidget" name="tab">
        <attribute name="title">
         <string>系统管理</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>设备管理</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <widget class="QWidget" name="page">
            <layout class="QVBoxLayout" name="verticalLayout">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTableWidget" name="tableWidget">
               <property name="editTriggers">
                <set>QAbstractItemView::NoEditTriggers</set>
               </property>
               <property name="selectionMode">
                <enum>QAbstractItemView::SingleSelection</enum>
               </property>
               <property name="selectionBehavior">
                <enum>QAbstractItemView::SelectRows</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="title">
                <string>分机%d只读参数</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_15">
                  <item>
                   <widget class="QLabel" name="label_14">
                    <property name="text">
                     <string>X频率</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_4">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="minimum">
                     <number>-9999</number>
                    </property>
                    <property name="maximum">
                     <number>9999</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout_16">
                  <item>
                   <widget class="QLabel" name="label_15">
                    <property name="text">
                     <string>X轴加速度特征值</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_7">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="suffix">
                     <string>m/s2</string>
                    </property>
                    <property name="minimum">
                     <number>-9999</number>
                    </property>
                    <property name="maximum">
                     <number>9999</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="2">
                 <layout class="QHBoxLayout" name="horizontalLayout_17">
                  <item>
                   <widget class="QLabel" name="label_16">
                    <property name="text">
                     <string>Y轴加速度特征值</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_10">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="suffix">
                     <string>m/s2</string>
                    </property>
                    <property name="minimum">
                     <number>-9999</number>
                    </property>
                    <property name="maximum">
                     <number>9999</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="3" rowspan="2">
                 <layout class="QHBoxLayout" name="horizontalLayout_18">
                  <item>
                   <widget class="QLabel" name="label_17">
                    <property name="text">
                     <string>Z轴加速度特征值</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_13">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="suffix">
                     <string>m/s2</string>
                    </property>
                    <property name="minimum">
                     <number>-9999</number>
                    </property>
                    <property name="maximum">
                     <number>9999</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="4" rowspan="2">
                 <layout class="QHBoxLayout" name="horizontalLayout_19">
                  <item>
                   <widget class="QLabel" name="label_18">
                    <property name="text">
                     <string>485通讯状况</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_40">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="minimum">
                     <number>0</number>
                    </property>
                    <property name="maximum">
                     <number>100</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0" rowspan="2">
                 <layout class="QHBoxLayout" name="horizontalLayout_20">
                  <item>
                   <widget class="QLabel" name="label_19">
                    <property name="text">
                     <string>Y频率</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_5">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="minimum">
                     <number>-9999</number>
                    </property>
                    <property name="maximum">
                     <number>9999</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="2" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout_21">
                  <item>
                   <widget class="QLabel" name="label_20">
                    <property name="text">
                     <string>X轴速度特征值</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_8">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="suffix">
                     <string>mm/s</string>
                    </property>
                    <property name="minimum">
                     <number>-9999</number>
                    </property>
                    <property name="maximum">
                     <number>9999</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="2" column="2">
                 <layout class="QHBoxLayout" name="horizontalLayout_22">
                  <item>
                   <widget class="QLabel" name="label_21">
                    <property name="text">
                     <string>Y轴速度特征值</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_11">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="suffix">
                     <string>mm/s</string>
                    </property>
                    <property name="minimum">
                     <number>-9999</number>
                    </property>
                    <property name="maximum">
                     <number>9999</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="2" column="3" rowspan="2">
                 <layout class="QHBoxLayout" name="horizontalLayout_23">
                  <item>
                   <widget class="QLabel" name="label_22">
                    <property name="text">
                     <string>Z轴速度特征值</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_14">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="suffix">
                     <string>mm/s</string>
                    </property>
                    <property name="minimum">
                     <number>-9999</number>
                    </property>
                    <property name="maximum">
                     <number>9999</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="3" column="2" rowspan="2">
                 <layout class="QHBoxLayout" name="horizontalLayout_24">
                  <item>
                   <widget class="QLabel" name="label_23">
                    <property name="text">
                     <string>Y轴位移特征值</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_12">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="suffix">
                     <string>um</string>
                    </property>
                    <property name="minimum">
                     <number>-9999</number>
                    </property>
                    <property name="maximum">
                     <number>9999</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="4" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_25">
                  <item>
                   <widget class="QLabel" name="label_24">
                    <property name="text">
                     <string>Z频率</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_6">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="minimum">
                     <number>-9999</number>
                    </property>
                    <property name="maximum">
                     <number>9999</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="4" column="1">
                 <layout class="QHBoxLayout" name="horizontalLayout_26">
                  <item>
                   <widget class="QLabel" name="label_25">
                    <property name="text">
                     <string>X轴位移特征值</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_9">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="suffix">
                     <string>um</string>
                    </property>
                    <property name="minimum">
                     <number>-9999</number>
                    </property>
                    <property name="maximum">
                     <number>9999</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="4" column="3">
                 <layout class="QHBoxLayout" name="horizontalLayout_27">
                  <item>
                   <widget class="QLabel" name="label_26">
                    <property name="text">
                     <string>Z轴位移特征值</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinBox_15">
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="suffix">
                     <string>um</string>
                    </property>
                    <property name="minimum">
                     <number>-9999</number>
                    </property>
                    <property name="maximum">
                     <number>9999</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="widget_log" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PollDeviceWidget</class>
   <extends>QWidget</extends>
   <header location="global">polldevicewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
